File: src/test/java/no/nav/dokgen/DevDokgen.java
package no.nav.dokgen;

import no.nav.dokgen.configuration.ApplicationConfig;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;
import org.springframework.boot.builder.SpringApplicationBuilder;

@SpringBootApplication(exclude = ErrorMvcAutoConfiguration.class)
public class DevDokgen {

    public static void main(String[] args) {
        new SpringApplicationBuilder(ApplicationConfig.class)
                .profiles("dev")
                .run(args);
    }

}

File: src/test/kotlin/no/nav/dokgen/DevDokgen.kt
package no.nav.dokgen

import no.nav.dokgen.configuration.ApplicationConfig
import org.springframework.boot.builder.SpringApplicationBuilder

object DevDokgen {
    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplicationBuilder(ApplicationConfig::class.java)
            .profiles("dev")
            .run(*args)
    }
}